Example-Crypto-REXX/RXCRYPT1.rexx

```diff
@@ -0,0 +1,2431 @@
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
+/*                                                                     */
+/*        EX 'HWI.HWIREST.REXX(RXCRYPT1))'                             */
+/*            '-D HWI.CRYPTO -S not-activated'                         */
+/*            data set is HWI.CRYPTO, run from TSO REXX,               */
+/*            query crypto attributes for not-activated LPARs          */
+/*            if successful will create HWI.CRYPTO(LOCAL)              */
+/*                                                                     */
+/* DEPENDENCIES:                                                       */
+/*     z16 - The crypto properties are supported on z16 processors     */
+/*           and above.                                                */
+/*                                                                     */
+/* NOTES:                                                              */
+/*     No recovery logic has been supplied in this sample.             */
+/*                                                                     */
+/* REFERENCE:                                                          */
+/*     See the z/OS MVS Programming: Callable Services for             */
+/*     High-Level Languages publication for more information           */
+/*     regarding the usage of HWIREST and JSON Parser APIs.            */
+/*                                                                     */
+/*                                                                     */
+/* END OF SPECIFICATIONS  * * * * * * * * * * * * * * * * * * * * * * **/
+
+MACLIB_DATASET = 'SYS1.MACLIB'
+
+TEMP_DATASET = '' /* required input via -D */
+
+TRUE = 1
+FALSE = 0
+
+hwiHostRC = 0     /* HWIHOST rc */
+HWIHOST_ON = FALSE
+PARSER_INIT = FALSE
+ISVREXX = FALSE  /* default to TSO/E, enable via -I */
+VERBOSE = FALSE  /* JSON parser specific, enabled via -V */
+localCPC = TRUE  /* default to lpars on the LOCAL CPC, change via -C */
+
+MemberName = 'LOCAL' /* default CPC */
+
+/* Properties constants */
+CRYPTOAUTH = 'crypto-activity-cpu-counter-authorization-control'
+ASSIGNED_DOMAINS_PROP = 'assigned-crypto-domains'
+ASSIGNED_CRYPTOS_PROP = 'assigned-cryptos'
+
+/*********************************/
+/* Get program args              */
+/*********************************/
+parse arg argString
+if GetArgs(argString) <> 0 then
+  exit -1
+
+/********************************************************/
+/* Ensure the output dataset specified actually exists  */
+/********************************************************/
+if VerifyDataSet(TEMP_DATASET) <> 0 then
+  do
+    errMsg = '** specified data set ('||TEMP_DATASET||') does not exist **'
+    exit fatalErrorAndCleanup(errMsg)
+  end
+
+/**********************************************/
+/* Ensure the requested LPAR status is valid  */
+/**********************************************/
+if VerifyStatus(ReqstdLPAR_Status) <> 0 then
```

Comment:
given the verifyStatus will always be successful ...this can just be a call to it w/out addition error throw

Reply:
Done